appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -434,12 +443,37 @@
                          registeredComponentName eventName)
                        #f))))
 
+       ;; For the HandlesEventDispatching interface
+       (define (dispatchMultiEvent eventName :: java.lang.String
```

Comment:
you said "The second block is like a normal event handler except you also have to give it the component that you want to use it for. So these two blocks behave the same"

let me ask a probably silly question:  when do you use the new block and when do you use the already existing block? or: what exactly is the advantage of the new block? Everything can be done currently without that new block?
new block:
![example](https://cloud.githubusercontent.com/assets/4552296/7106622/63b0c404-e115-11e4-932a-ac6b40e1ceea.png)
current block:
![unbenannt](https://cloud.githubusercontent.com/assets/2848474/7107119/efcb1cea-e116-11e4-9f0b-99304de82920.PNG)


Taifun